CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+# How to contribute to U.S. Digital Response projects
+
+Everyone is welcome to contribute, and we value everybody's contribution. Code is not the only
+way to help the community. Answering questions, helping others, reaching out, and improving
+documentation are all immensely valuable to the community.
+
+
+## You can contribute in so many ways!
+
+There are several ways you can contribute to this repository:
+
+- Enhancing and/or fixing outstanding issues with the existing code
+- Contributing useful improvements, clarifications, and updates to existing documentation
+- Submitting [issues](https://github.com/usdigitalresponse/grants-ingest/issues/new/choose)
+related to bugs or desired new features
+- Reporting security vulnerabilities
+
+_All are equally valuable to the community._
+
+We also onboard new volunteers to USDR projects in general at https://www.usdigitalresponse.org/volunteer.
+Please sign up if you’d like to help on other projects.
+
+
+## Submitting a new issue or feature request
+
+Do your best to follow these guidelines when submitting an issue or a feature request. It will
+make it easier for us to come back to you quickly and with good feedback.
+
+First, please check whether your issue or feature request has already been submitted by searching
+our [open issues](https://github.com/usdigitalresponse/grants-ingest/issues?q=is%3Aopen+is%3Aissue).
+
+
+### Bugs
+
+If you find an [open bug](https://github.com/usdigitalresponse/grants-ingest/issues?q=is%3Aopen+is%3Aissue+label%3Abug)
+that sounds like your issue, please consider adding a comment to the existing issue with details
+that will help us better understand the problem.
+
+To submit something new, please fill out and submit
+[this form](https://github.com/usdigitalresponse/grants-ingest/issues/new?template=default_issue.yml&title=%5BBug%5D%3A+).
+
+
+### Security vulnerabilities
+
+Please fill out and submit [this form](https://github.com/usdigitalresponse/grants-ingest/security/advisories/new)
+if you believe you have discovered a security-related problem.
+
+
+### New Features
+
+A world-class feature request addresses the following points:
+
+1. Motivation first:
+    - Is it related to a problem/frustration with the current system? If so, ease explain why.
+    - Is it related to something you would need for a project? We'd love to hear about it!
+    - Is it something you worked on and think could benefit the community? Awesome! Tell us
+    what problem it solved for you.
+    - Do you think you have a neat idea and want to share it? We love those too! Tell us your
+    use-case and what value it might bring to the project.
```

Comment:
```suggestion
    - Do you think you have a neat idea and want to share it? 
    We love those too! 
    Tell us your use-case and what value it might bring to the project.
```